helm/flannel-operator-chart/values.yaml

```diff
@@ -1,6 +1,30 @@
-clusterRoleBindingName: flannel-operator
-clusterRoleBindingNamePSP: flannel-operator-psp
-clusterRoleName: flannel-operator
-clusterRoleNamePSP: flannel-operator-psp
-namespace: giantswarm
-pspName: flannel-operator-psp
+image:
+  registry: "quay.io"
+  repository: "giantswarm/flannel-operator"
+  tag: "[[ .SHA ]]"
+pod:
+  user:
+    id: 0
+  group:
+    id: 1000
+# Resource names are truncated to 47 characters. Kubernetes allows 63 characters
+# limit for resource names. When pods for deployments are created they have
+# additional 16 characters suffix, e.g. "-957c9d6ff-pkzgw" and we want to have
+# room for those suffixes.
+#
+# NOTE: All values under resource key need to be used with `tpl` to render them
+# correctly in the templates. This is because helm doesn't template values.yaml
+# file and it has to be a valid json. Example usage:
+#
+#     {{ tpl .Values.resource.default.name . }}.
+#
+resource:
+  configMap:
+    name: '{{ .Release.Name | replace "." "-" | trunc 47 }}-configmap'
```

Comment:
I'm a noob here, why do we need this here? why isn't it like a static thing like  `flannel-operator-configmap` ?

Reply:
Mostly just standardisation - one thing I've found whilst touching all these repos is that there are a lot of different approaches to the same thing. Pawel has started with the above approach in part due to VOO (I believe).

Reply:
yes, we might be running 2 flannel operators in VOO for example